static/css/zamboni/reviewers.less

```diff
@@ -1129,9 +1129,33 @@ pre.history-comment {
     font-style: italic;
 }
 
-.history-notes {
-    max-width: 370px;
+.history-notes, .history-comment {
     overflow: auto;
+    max-height: 50em;
+    /* hack to show gradients indicating that there is some scrolling to do,
+       in case the browser would decide to hide it despite overflow-y: scroll;
+       See https://lea.verou.me/blog/2012/04/background-attachment-local/
+    */
+    background:
+        /* Shadow covers */
+        linear-gradient(white 30%, rgba(255,255,255,0)),
+        linear-gradient(rgba(255,255,255,0), white 70%) 0 100%,
+
+        /* Shadows */
+        radial-gradient(50% 0, farthest-side, rgba(0,0,0,.2), rgba(0,0,0,0)),
+        radial-gradient(50% 100%,farthest-side, rgba(0,0,0,.2), rgba(0,0,0,0)) 0 100%;
+    background:
+        /* Shadow covers */
+        linear-gradient(white 30%, rgba(255,255,255,0)),
+        linear-gradient(rgba(255,255,255,0), white 70%) 0 100%,
+
+        /* Shadows */
+        radial-gradient(farthest-side at 50% 0, rgba(0,0,0,.2), rgba(0,0,0,0)),
+        radial-gradient(farthest-side at 50% 100%, rgba(0,0,0,.2), rgba(0,0,0,0)) 0 100%;
```

Comment:
duplicated rules?

Reply:
It's not strictly duplicated, there are subtle differences. The first background is for compatibility with older syntax (it's ignored by browsers supporting the newer syntax, the second rule overwrites it). I don't think we care about backwards-compatibility here though, so I'll remove it.

Reply:
ah, the first two lines (`linear-gradient`) are identical in each (right?), but I overlooked the difference in `radial-gradient'

Reply:
Yes, only the `radial-gradient` syntax differs (the second one using the more modern `at` version... well, more modern in 2012 when the article was written), but the entire `background` needs to be repeated for the compatibility trick to work.

I've removed the old version in https://github.com/mozilla/addons-server/pull/23267/commits/a63ea49d33ddccc250aadd249965ec0eb7e4f5f7